### PR TITLE
SW-5167 Fix country/region selection dependency

### PR DESCRIPTION
**Pull request by karthikbtf**
- country alone is selectable
- region is applied from country
- region is read-only

### CODE REVIEW

File: src/components/ProjectField/RegionSelect.tsx [hunk 1]
```diff
@@ -2,19 +2,24 @@ import { useMemo } from 'react';
...
+
   const options = useMemo(
     (): DropdownItem[] =>
-      REGIONS.map((_value) => ({
-        label: getRegion(_value),
-        value: _value,
-      })),
-    []
+      activeLocale
+        ? REGIONS().map(({ region, label: strLabel }) => ({
```
Comment (nickgraz) [inline src/components/ProjectField/RegionSelect.tsx:17]: 🙏

**Review by nickgraz — APPROVED** [review]
Looks great!

**Comment by karthikbtf** [comment]
> > Looks great!
> 
> > Looks great!
> 
> Thanks 🙏 I have some open UX questions in the [Jira](https://terraformation.atlassian.net/browse/SW-5167) for follow up.

Latest change based on Jira feedback
- Swap region and country in position
- Select region first, disable country until region is selected
- Country dropdown list narrowed by region selection

**Comment by karthikbtf** [comment]
> > > Looks great!
> > 
> > 
> > > Looks great!
> > 
> > 
> > Thanks 🙏 I have some open UX questions in the [Jira](https://terraformation.atlassian.net/browse/SW-5167) for follow up.
> 
> Latest change based on Jira feedback
> 
> * Swap region and country in position
> * Select region first, disable country until region is selected
> * Country dropdown list narrowed by region selection

Final changes as per Jira.
- country alone is selectable
- region is applied from country
- region is read-only

**Review by nickgraz — APPROVED** [review]
Looks good

**Pull request merged**